### ISSUE TITLE
开启后无法联网问题

### ISSUE THREAD

**Issue by yuanlun1983** [garypang13/luci-app-dnsfilter#34]
用的是
ImmortalWrt5.15官方固件，启动luci-app-dnsfilter成功后导致所有设备不能联网，甚至连wifi都不能连接，关了就正常，不知道什么原因。上网环境是无线中继，会不会和这个有关？

**Comment by zow2023** [comment]
是的 我也遇见一样的问题  lean的源码编译的AX6的固件  5.4内核的不会 5.10的就会   希望有人能来指点下哪里出了问题，。

**Comment by yuanlun1983** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by ZhongMoDao** [comment]
> 是的 我也遇见一样的问题 lean的源码编译的AX6的固件 5.4内核的不会 5.10的就会 希望有人能来指点下哪里出了问题，。

因为firewall4冲突

**Comment by zow2023** [comment]
> > 是的 我也遇见一样的问题 lean的源码编译的AX6的固件 5.4内核的不会 5.10的就会 希望有人能来指点下哪里出了问题，。
> 
> 因为firewall4冲突

 我编译的时候 并没有选择firewall4 软件包已安装的软件显示也没有firewall4的。
